### PR TITLE
Bricklayers unite!

### PR DESCRIPTION
**Pull request by rhrab**
Now we can build a house! Hurray!

TLDR: added map files and finished with bricks.

### CODE REVIEW

File: source/main.cpp [hunk 1]
```diff
@@ -107,25 +118,29 @@ int main(int argc, char* argv[])
...
+				}
 			}
 		}
 
 		env.events.EmitEvent("TickUpdate");
 
 		env.events.EmitEvent("MarkDestroy");
 		//TODO: solve not work because iterator invalidates after call delete this and unsubscribe
-		// Destroy all dead objects
+		// Destroy all "dead" objects (excluding mapBlocks)
```
Comment (roma4004) [inline source/main.cpp:133]: comment not actual due last refactoring

File: headers/Environment.h [hunk 2]
```diff
@@ -66,10 +66,8 @@ struct Environment
 	void SetPixel(int x, int y, int color) const;
 
 	MouseButtons mouseButtons;
-	std::vector<BaseObj*> allPawns;
+	std::vector<std::unique_ptr<BaseObj>> allPawns;
 	std::list<BaseObj*> pawnsToDestroy;
```
Comment (rhrab) [inline headers/Environment.h:70]: redundant . need to be deleted

File: headers/Bullet.h [hunk 3]
```diff
@@ -8,24 +8,25 @@ struct Environment;
...
+	Bullet(const Point& pos, int width, int height, int color, float speed, Direction direction, int health,
+		   Environment* env);
 
 	~Bullet() override;
 
-	void Move(Environment* env) override;
-	void Draw(const Environment* env) const override;
+	void Move() override;
+	void Draw() const override;
 	void KeyboardEvensHandlers(Environment& env, Uint32 eventType, SDL_Keycode key) override;
```
Comment (rhrab) [inline headers/Bullet.h:18]: need to fix arguments. (Must be: void KeyboardEvensHandlers(Uint32 eventType, SDL_Keycode key) override;

Comment (roma4004) [reply headers/Bullet.h]: now using internal field _env and for bullet method was marked as deleted

**Pull request merged**